### ISSUE TITLE
Regression 4.0.1 -> 4.0.2

### ISSUE THREAD

**Issue by simlu** [addaleax/lzma-native#71]
After updating from 4.0.1 to 4.0.2 I get an error:

```
/var/lang/bin/node: symbol lookup error: /user/project/node_modules/lzma-native/binding-v4.0.2-node-v57-linux-x64/lzma_native.node: undefined symbol: _ZN2v87Isolate19CheckMemoryPressureEv
```

Any idea what's going on here? 

Thank you very much! L~

**Comment by addaleax** [comment]
@simlu Can you share more information? In particular, which version of Node.js are you using?

**Comment by simlu** [comment]
If you're looking for a way to easily reproduce, we are running docker image `lambci/lambda:build-nodejs8.10`.

Let me know if desired and I can produce a more complete example!

**Comment by simlu** [comment]
@addaleax Thank you for the quick investigation!

Yeah, unfortunately we will be stuck on 8.10 for a while. AWS doesn't update lambda versions very often. We're ok with waiting for the next lzma-native release though.

Thanks again very much for your help! L~

**Comment by addaleax** [comment]
Okay, just released v4.0.3 – let me know if you run into any trouble :)
